example/resources/langs/en.json

```diff
@@ -1,6 +1,7 @@
 {
     "title": "Hello",
     "msg": "Hello {} in the {} world ",
+    "msg_named": "This text is displayed in {{lang}} language",
```

Comment:
Easy localization are written in the {{lang}} language

Reply:
> Easy localization are written in the {{lang}} language

about this, we can do 
`"msg_named": " {} are written in the {{lang}} language",`